docs/source/api.rst

```diff
@@ -1,17 +1,105 @@
 API Reference
 =============
 
-.. contents::
-    :depth: 2
-
-.. models:
-Models
-------
-.. automodule:: orchestra.models
-    :members:
-
-.. task_lifecycle:
-Task Lifecycle
---------------
-.. automodule:: orchestra.utils.task_lifecycle
-    :members:
+Client API
+----------
+
```

Comment:
add a note that they have to sign the request using http signatures (http://tools.ietf.org/html/draft-cavage-http-signatures-03)

See an example in `_make_api_request` of https://github.com/unlimitedlabs/orchestra/blob/master/orchestra/orchestra_api.py#L29